corehq/apps/reports/api.py

```diff
@@ -41,7 +41,7 @@ def get_data(self, start=None, limit=None):
         [{'village': 'Mazu', 'births': 30, 'deaths': 28},{...}]
         """
 
-        return {}
+        return [{}]
```

Comment:
has this just always been wrong or is it related to the change in some way?

Reply:
my guess based on the other custom report changes: previously there were some instances that returned a dict if there was only one result that have now been converted to a list (where is this change?)

Reply:
> where is this change?

my guess based on further digging is that's the crux of this change: https://github.com/dimagi/sql-agg/pull/56/

Reply:
this was just to reflect what is expected of classes inheriting from this abstract class, which was indeed not correct. You can see the example in the documentation right above it.